### PR TITLE
[SYNPY-1370] migrate to google style

### PR DESCRIPTION
**Pull request by jaymedina**
This PR will update the following documentation:

- [ ] Small few left: https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/synapseclient/annotations.py 
- [ ] Evaluation - The section at the top of the file should be changed from RST into MD format. See [Table for an example](https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/synapseclient/table.py): https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/synapseclient/evaluation.py 
- [ ] Wiki The section at the top of the file should be changed from RST into MD format. See [Table for an example](https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/synapseclient/table.py):: https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/synapseclient/wiki.py 
- [ ] https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/synapseclient/core/retry.py 
Should also add this to https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/docs/reference/core.md
- [ ] https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/synapseclient/core/version_check.py 
- [ ] https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/synapseclient/core/upload/multipart_upload.py 
- [ ] https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/synapseutils/sync.py

### CODE REVIEW

File: synapseclient/evaluation.py [hunk 1]
```diff
@@ -34,28 +32,26 @@
...
-- :py:func:`synapseclient.Synapse.getSubmissions`
-- :py:func:`synapseclient.Synapse.getSubmission`
-- :py:func:`synapseclient.Synapse.getSubmissionStatus`
-- :py:func:`synapseclient.Synapse.getPermissions`
-- :py:func:`synapseclient.Synapse.setPermissions`
-
-~~~~~~~~~~
-Evaluation
-~~~~~~~~~~
+- `synapseclient.Synapse.getEvaluation`
```
Comment (BryanFauble) [inline synapseclient/evaluation.py:44]: These should be links using cross-reference: https://mkdocstrings.github.io/usage/#cross-references

To auto link to the applicable function in the python docs. This is similar to what you've got in the file:
```suggestion
- [synapseclient.Synapse.getEvaluation][]
```

Test this out. But I think this was the proper format you can use if you don't want to display different text.


You can use this same format above as well. I don't thing we need to show all the function text in a `code block`

**Comment by BryanFauble** [comment]
Create a branch within the https://github.com/Sage-Bionetworks/synapsePythonClient/tree/develop repo itself. If you fork the repo and make the pull request you 1) Won't be able to run integration tests and 2) Submit the sonarCloud scan results.

**Comment by jaymedina** [comment]
Thanks @BryanFauble - Here are the docs that mention forking the repo + adding GH secrets to the fork to run the integration tests:
* https://sagebionetworks.jira.com/wiki/spaces/SYNPY/pages/65503359/Python+Client+Developer+Bootstrap#Retrieve-the-code-base-from-Github
* https://sagebionetworks.jira.com/wiki/spaces/SYNPY/pages/65503359/Python+Client+Developer+Bootstrap#GitHub-Actions
* https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/CONTRIBUTING.md#fork-and-clone-this-repository

I'll remove the secrets from my fork and proceed with a branch under the main repo, then.

**Comment by thomasyu888** [comment]
Thanks for raising those @jaymedina - sorry about that!  That used to be the protocol, but I've since changed it for convenience, as we are the main maintainers of the package - we will spend some time updating the documentation you linked out!  The permissions is set up so that only admins can directly push to `master` or `develop` branch

The contributing docs should probably be explicit about it being a guide for "external to Sage" contribution.

**Pull request closed without merging**